packages-next/fields/src/types/image/views/Field.tsx

```diff
@@ -204,12 +202,7 @@ function ImgView({
       {errorMessage === undefined ? (
         value.kind === 'from-server' ? (
           <ImageWrapper>
-            <NextImage
-              height={value.data.height}
-              width={value.data.width}
-              src={value.data.src}
-              alt={field.path}
-            />
+            <img css={{ width: '100%' }} src={value.data.src} alt={field.path} />
```

Comment:
This change looks valid (?), but orthogonal to the cloud stuff. Can we put it up as a separate PR for review with its own changeset.

Reply:
This needs to be done in order for the cloud images to work in the admin UI